core/src/test/java/io/github/tcdl/impl/ResponderServerImplTest.java

```diff
@@ -63,17 +64,17 @@ public void testResponderServerProcessSuccess() throws Exception {
 
     @Test(expected = NullPointerException.class)
     public void testResponderServerProcessErrorNoHandler() throws Exception {
-        ResponderServerImpl.create(TestUtils.getSimpleNamespace(), messageTemplate, msbContext, null);
+        msbContext.getObjectFactory().createResponderServer(TestUtils.getSimpleNamespace(), messageTemplate, null);
     }
 
     @Test
     public void testResponderServerProcessWithError() throws Exception {
         Exception error = new Exception();
-        ResponderServerImpl.RequestHandler handler = (request, responder) -> { throw error; };
+        ResponderServer.RequestHandler handler = (request, responder) -> { throw error; };
 
-        ResponderServerImpl responderServer = (ResponderServerImpl) ResponderServerImpl
-                .create(TestUtils.getSimpleNamespace(), messageTemplate, msbContext, handler)
-                .listen();
+        ResponderServerImpl responderServer = ResponderServerImpl
```

Comment:
no as we use here packege-private onResponder(). But one line above we can change:
 ResponderServerImpl.RequestHandler handler = (request, responder) -> { throw error; };
->
 ResponderServer.RequestHandler handler = (request, responder) -> { throw error; };